### PR TITLE
Add CodeTriage button

### LINKED ISSUE #639: Add link to CodeTriage to repo goals for reminders
### Is your feature request related to a problem? Please describe.

Code Triage already exist and is doing great work. I discovered if you add the github owner and repo to the end of the codetriage.com url it will navigate to a project for triaging or have you create it in the platform.  

### Describe the solution you'd like

Add a link or button so users can track the project and get emails when new issues are available to triage.

i.e. https://codetriage.com/mutual/beybot

### Describe alternatives you've considered

updates

### PR DESCRIPTION
**Pull request by yg**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

#639

## Related Tickets & Documents

Closes #639

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

Screenshot from 2020-06-18 12-39-47

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by yg** [comment]
@bdougie I need help if we need to design like the mockup you show!

File: src/components/Repository.js [hunk 1]
```diff
@@ -96,6 +97,15 @@ function Repository({match}) {
                 {repository && repository.languages.totalCount > languagesShown && `+${repository.languages.totalCount - languagesShown} languages`}
               </span>
             </div>
+            <Flex>
+              {repository && (
+                <a rel="noreferrer" target="_blank" href={`https://codetriage.com/${nameWithOwner}`}>
+                  <Button primary>
+                    Watch on Codetriage
```
Comment (sujitagarwal) [inline src/components/Repository.js:104]: I think it'll be a good idea to follow the Casing convention of their brand name, ie. CodeTriage and not Codetriage.

image

Comment (yg) [reply src/components/Repository.js:104]: @sujitagarwal Good catch! I will update it! :+1:

Comment (yg) [inline src/components/Repository.js:104]:
```suggestion
                    Watch on CodeTriage
```

**Comment by yg** [comment]
@bdougie Still there is some whitespace!

Screenshot

**Comment by yg** [comment]
@bdougie I think it is better now! But I'm not sure what the API returns!

Screenshot

**Comment by yg** [comment]
> Can we remove the illustration for now, or `display:none` it for any smaller than [full screen DESKTOP](https://github.com/open-sauced/open-sauced/blob/master/src/styles/Illustration.js#L12).

`display:none` will be fine

**Comment by yg** [comment]
Added contribution count!

https://www.loom.com/share/c0cb67a40e2a41398b6cb8cf3dd9ac36

**Comment by yg** [comment]
@bdougie How can I handle error for this https://serve.onegraph.com/short/QBF7TG

Still, there is a Repo but it has tons of contributors! Example (**torvalds/linux**)

Screenshot

**Comment by filiptronicek** [comment]
What about we do the contributors the way we do issue participants: with the stacked avatars. It wouldn't be nice to forget those, who have contributed to the repos, even if in smaller amounts! **Other than that, I really like this implementation.** @yg

**Comment by yg** [comment]
> What about we do the contributors the way we do issue participants: with the stacked avatars. It wouldn't be nice to forget those, who have contributed to the repos, even if in smaller amounts! **Other than that, I really like this implementation.** @yg

@filiptronicek Stacked Avatars looks nice, but here I think it is not suitable! Even GitHub hasn't added it to the Repo meta section!

While stacked we can see a lot of whitespace on the right side.

Screenshot

**Comment by filiptronicek** [comment]
> 
> 
> > What about we do the contributors the way we do issue participants: with the stacked avatars. It wouldn't be nice to forget those, who have contributed to the repos, even if in smaller amounts! **Other than that, I really like this implementation.** @yg
> 
> @filiptronicek Stacked Avatars looks nice, but here I think it is not suitable! Even GitHub hasn't added it to the Repo meta section!
> 
> While stacked we can see a lot of whitespace on the right side.
> 
> Screenshot

What about we do something like GitHub, just to mention there are more contributors than this? @yg

**Comment by yg** [comment]
@filiptronicek Currently I'm facing issue on fetching `totalCount`, for now, I just displayed something like this!

Screenshot

PS: We removed bots :robot:

**Comment by filiptronicek** [comment]
Awesome, this is good!

**Comment by filiptronicek** [comment]
What about adding a profile link to their respective GH profiles? (Probably launching in a new tab) @yg

**Comment by filiptronicek** [comment]
> 
> 
> > What about adding a profile link to their respective GH profiles? (Probably launching in a new tab) @yg
> 
> @filiptronicek Just added it [c3fc763](https://github.com/open-sauced/open-sauced/commit/c3fc763c570e3efe666ff8f68a8f1f8b0d2107ff) 🎉

Sorry for that, had my old browser cache.

**Comment by bdougie** [comment]
> Still, there is a Repo but it has tons of contributors! Example (torvalds/linux)

@yg, for now, I don't think we need to address this and presenting the error on large repos is fine, the chances of a user contributing to Linux via open sauced is close to 0. 

If you were to work on this, I see us handling like so. 

Leverage a fallback query if the query **with** contributors fails then use a retry with another query **without** contributors. 

Apollo-Link has a [retry plugin](https://www.npmjs.com/package/apollo-link-retry). I don't think we could use that for this option, but we could model a new service specific for this project that does just that.. 

Something the combines apollo-link-retry and [apollo-link-error](https://www.apollographql.com/docs/link/links/error/) to something like query-fallback-on-error (not implemented).

**Comment by yg** [comment]
> Apollo-Link has a [retry plugin](https://www.npmjs.com/package/apollo-link-retry). I don't think we could use that for this option, but we could model a new service specific for this project that does just that..
> 
> Something the combines apollo-link-retry and [apollo-link-error](https://www.apollographql.com/docs/link/links/error/) to something like query-fallback-on-error (not implemented).

Ooo! this looks complex for me 🤯

**Comment by filiptronicek** [comment]
If you want I could make a simple REST API just to get like the 5 contributors, who contributed the most and then a total number of them, shouldn't be that hard. @yg

**Comment by yg** [comment]
> If you want I could make a simple REST API just to get like the 5 contributors, who contributed the most and then a total number of them, shouldn't be that hard. @yg

@filiptronicek I think implementing REST will increase complexity!

@bdougie WDYT?

**Comment by bdougie** [comment]
>> If you want I could make a simple REST API just to get like the 5 contributors, who contributed the most and then a total number of them, shouldn't be that hard. @yg

> @filiptronicek I think implementing REST will increase complexity!

> @bdougie WDYT?

Yep, I think abstracting what I put into a simple rest service, similar to the status code service would work. But I am also concerned at the complexity and would prefer to punt on this until it is a real problem and an actually repo that you can contribute to is blocked.

**Comment by yg** [comment]
@bdougie I think we can handle it later for big repos!

**Pull request merged**